[1.7.10] Concurrent modification exception while drawing tooltips

```
[07:44:09] [Client thread/FATAL]: Reported exception thrown!
net.minecraft.util.ReportedException: Rendering screen
	at net.minecraft.client.renderer.EntityRenderer.func_78480_b(EntityRenderer.java:1092) ~[blt.class:?]
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1001) ~[bao.class:?]
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:898) [bao.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:148) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_73]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_73]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_73]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_73]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
Caused by: java.util.ConcurrentModificationException
	at java.util.HashMap$HashIterator.nextNode(Unknown Source) ~[?:1.8.0_73]
	at java.util.HashMap$EntryIterator.next(Unknown Source) ~[?:1.8.0_73]
	at java.util.HashMap$EntryIterator.next(Unknown Source) ~[?:1.8.0_73]
	at java.util.Hashtable.putAll(Unknown Source) ~[?:1.8.0_73]
	at cpw.mods.fml.common.registry.LanguageRegistry.injectLanguage(LanguageRegistry.java:253) ~[LanguageRegistry.class:?]
	at gregtech.api.util.GT_LanguageManager.addStringLocalization(GT_LanguageManager.java:28) ~[GT_LanguageManager.class:?]
	at gregtech.api.util.GT_LanguageManager.addStringLocalization(GT_LanguageManager.java:21) ~[GT_LanguageManager.class:?]
	at gregtech.common.items.GT_IntegratedCircuit_Item.addAdditionalToolTips(GT_IntegratedCircuit_Item.java:82) ~[GT_IntegratedCircuit_Item.class:?]
	at gregtech.api.items.GT_Generic_Item.func_77624_a(GT_Generic_Item.java:98) ~[GT_Generic_Item.class:?]
	at net.minecraft.item.ItemStack.func_82840_a(ItemStack.java:525) ~[add.class:?]
	at forestry.core.gui.widgets.ItemStackWidgetBase.getToolTip(ItemStackWidgetBase.java:41) ~[ItemStackWidgetBase.class:?]
	at forestry.core.gui.GuiUtil.drawToolTips(GuiUtil.java:133) ~[GuiUtil.class:?]
```

`gregtech-5.09.25`
`forestry_1.7.10-4.2.16.64`
`forge-1.7.10-10.13.4.1614`

Happens occasionally when opening a Forestry Worktable that contains an Integrated Circuit, either in its storage or as a stored recipe. May be either Forestry's or Gregtech's fault - can't tell at this point.

The CME indicates that another thread has modified the language registry at the same time as the client thread attempted to modify it. This means I should eventually get the same error but on the side of the other thread. I will update this issue when it happens.

<sub>Or if we're lucky, someone else has the other half of this error and finds this issue.</sub>

--- a/src/main/java/gregtech/api/util/GT_LanguageManager.java
+++ b/src/main/java/gregtech/api/util/GT_LanguageManager.java
@@ -22,7 +22,7 @@ public class GT_LanguageManager {
         return addStringLocalization(aKey, aEnglish, true);
     }
 
-    public static String addStringLocalization(String aKey, String aEnglish, boolean aWriteIntoLangFile) {
+    public static synchronized String addStringLocalization(String aKey, String aEnglish, boolean aWriteIntoLangFile) {
         if (aKey == null) return E;
         if (aWriteIntoLangFile){ aEnglish = writeToLangFile(aKey, aEnglish);
         if(!LANGMAP.containsKey(aKey)){

